### PR TITLE
changed endpoints for `setup/repos` APIs

### PR DESCRIPTION
**Pull request by MuskanPaliwal**
-Created another API for fetching bitbucket setup repos from db, hence changed the end-points here as well
-The endpoint will be `/bitbucket/setup/repos` for bitbucket and `/github/setup/repos` for github
-The bitbucket API expects frontend to send `workspace_slug` key-value pair in request.body.

### CODE REVIEW

**Review by tapishr — DISMISSED** [review]
Looking good to me

File: scripts/contentScript.js [hunk 1]
```diff
@@ -138,10 +138,17 @@ async function sha256(value) {
 }
 
 // for showing all tracked/ untrack pr in a organization
-async function getTrackedRepos(orgName, userId) {
+async function getTrackedRepos(orgName, userId, is_github) {
```
Comment (avikalpg) [inline scripts/contentScript.js:141]: I think the `getTrackedRepos` function is currently only called for the Github case at the moment. 
@ajayVibinex Please confirm.

File: scripts/contentScript.js [hunk 2]
```diff
@@ -416,7 +423,7 @@ const orchestrator = (tab_url, websiteUrl, userId) => {
 				// for woking on this url https://github.com/Alokit-Innovations or https://github.com/orgs/Alokit-Innovations/repositories?type=all type 
 				const org_name = (urlObj[3] === "orgs") ? urlObj[4] : urlObj[3];
 				const body = { "org": org_name, "userId": userId, "is_github": true }
-				const url = `${backendUrl}/setup/repos`;
+				const url = `${backendUrl}/github/setup/repos`;
 				const response = await apiCall(url, body);
```
Comment (avikalpg) [inline scripts/contentScript.js:427]: Kind of makes the `getTrackedRepos` function pointless.

**Review by avikalpg — CHANGES_REQUESTED** [review]

File: scripts/contentScript.js (hunk 1 above)
Comment (ajayVibinex) [reply scripts/contentScript.js:141]: Yes, @avikalpg its for gitHub only. basically this is for , when the loading finishes inside a repo so to check weather it's tracked or not. We can also add it to bitbucket.

File: scripts/contentScript.js [hunk 3]
```diff
@@ -426,8 +433,8 @@ const orchestrator = (tab_url, websiteUrl, userId) => {
 			// for showing tracked repo of a organization 
 			if (urlObj[4] === 'workspace' && urlObj[5] === 'repositories') {
 				const workspace_slug = urlObj[3];
-				const body = { "org": workspace_slug, "userId": userId, "is_github": false }
-				const url = `${backendUrl}/setup/repos`;
+				const body = { "workspace_slug": workspace_slug, "userId": userId, "is_github": false }
+				const url = `${backendUrl}/bitbucket/setup/repos`;
 
 				const response = await apiCall(url, body);
 				updateTrackedReposInBitbucketOrg(response.repos, websiteUrl);
```
Comment (ajayVibinex) [inline scripts/contentScript.js:440]: Reviewed it , apart from changing api endpoints, you changed the function getTracked repos and now it'll will work for both gitHub and bitBucket if api is right but it's not called for bitBucket.

File: scripts/contentScript.js (hunk 1 above)
Comment (MuskanPaliwal) [reply scripts/contentScript.js:141]: So I am not supposed to remove the function right?

File: scripts/contentScript.js (hunk 3 above)
Comment (MuskanPaliwal) [reply scripts/contentScript.js:440]: Got it

File: scripts/contentScript.js (hunk 1 above)
Comment (ajayVibinex) [reply scripts/contentScript.js:141]: Yes don't remove this function it's important for showing the reload and add button !

**Pull request merged**